Driverless AI/scoring-pipeline-deployment-to-apache-nifi/scoring-pipeline-deployment-to-apache-nifi.md

```diff
@@ -0,0 +1,352 @@
+
+![cylinder-diagram-1](assets/cylinder-diagram-1.jpg)
+Figure: Hydraulic Test Rig General Cylinder Diagram
+
+The Hydraulic Test Rig consists of a primary and secondary cooling filtration circuit with pumps that deliver flow and pressure to the oil tank. The oil tank box at the bottom. There is a pressure relief control valve for controlling the rising and falling flows. There is a pressure gauge for measuring the pressure. 
+
+### Resources
+
+[1] H2O.ai Community AI Glossary: [Machine Learning Model Deployment](https://www.h2o.ai/community/glossary/machine-learning-model-deployment-productionization-productionizing-machine-learning-models)
+
+[2] H2O.ai Community AI Glossary: [Feature Engineering](https://www.h2o.ai/community/glossary/feature-engineering-data-transformation)
+
+[3] H2O.ai Community AI Glossary: [Automatic Machine Learning (AutoML)](https://www.h2o.ai/community/glossary/automatic-machine-learning-automl)
+
+[4] H2O.ai Community AI Glossary: [Machine Learning Model](https://www.h2o.ai/community/glossary/machine-learning-model)
+
+[5] H2O.ai Community AI Glossary: [Scoring Pipeline](https://www.h2o.ai/community/glossary/scoring-pipeline)
+
+[6] H2O.ai Community AI Glossary: [Model Object, Optimized (MOJO) Scoring Pipeline](https://www.h2o.ai/community/glossary/model-object-optimized-mojo)
+
+[7] [SAVERY - HYDRAULIC TEST RIGS AND BENCHES](https://www.savery.co.uk/systems/test-benches)
+
+[8] [HYDROTECHNIK - Flow and Temperature Testing Components](https://www.hydrotechnik.co.uk/flow-and-temperature-hydraulic-test-bed)
+
+## Prerequisites
+
+- Skilled in Java Object Oriented Programming
+- Driverless AI Environment
+- Driverless AI License
+    - 21 day trial license
+    - It is needed for using the MOJO2 Java Runtime API to execute the MOJO Scoring Pipeline in Apache NiFi
+    - If you need to purchase a Driverless AI license, reach out to our sales team via the [contact us form](https://www.h2o.ai/company/contact/).
+- Basic knowledge of Driverless AI or doing the following tutorials:
```

Comment:
rewrite this sentence: "Basic knowledge of Driverless AI or doing the following tutorials:"

to this: "Basic knowledge of Driverless AI or completion of the following tutorials:"